main/src/ca/uwaterloo/flix/language/phase/jvm/JvmOps.scala

```diff
@@ -246,8 +246,7 @@ object JvmOps {
   }
 
   private def getNamespaceName(ns: List[String]): JvmName = {
-    val name = JvmName.mkClassName("Ns")
-    JvmName(ns, name)
+    JvmName(ns, "Export")
```

Comment:
Do we want it to reside in Export? Why can it not just be where we expect?

Reply:
I didn't know what to do about the Root namespace edge case

Reply:
Hmm. True. You could disallow it the same way you disallow polymorphism etc.